articles/article-125-euler-path.md

```diff
@@ -0,0 +1,18 @@
+<!-- BEGIN TITLE -->
+# How to navigate in Königsberg
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+The city of Königsberg is split by a river into 4 land masses. Those are connected by 7 lovely bridges that invite to a walk. The best route is of course the one which crosses each bridge exactly once.
+But what is this route? Leonard Euler showed that unfortunately no such route exists in Königsberg, thereby he introduced the world to graphtheory, a tool to study one of the most important structures in computer science.
```

Comment:
Leonard Euler showed that unfortunately no such route exists in Königsberg, thereby introducing to the world graph theory, a tool to study one of the most important structures in computer science.

changed the sentence to not make it a run-on sentence which would have to be split in two.